members/models/person.py

```diff
@@ -75,5 +89,43 @@ def age_years(self):
 
     def firstname(self):
         return self.name.partition(' ')[0]
+
+    def update_dawa_data(self):
+        if(self.dawa_id is None or
+           self.latitude is None or
+           self.longitude is None or
+           self.municipality is None):
+            addressID = 0
+            dist = 0
+            req = 'https://dawa.aws.dk/datavask/adresser?betegnelse='
+            req += quote_plus(self.addressWithZip())
+            try:
+                washed = json.loads(requests.get(req).text)
+                addressID = washed['resultater'][0]['adresse']['id']
+                dist = washed['resultater'][0]['vaskeresultat']['afstand']
+            except Exception as error:
+                print("Couldn't find addressID for " + self.name)
+                print("Error " + str(error))
+            if (addressID != 0 and dist < 10):
+                try:
+                    req = 'https://dawa.aws.dk/adresser/' + addressID + "?format=geojson"
+                    address = json.loads(requests.get(req).text)
+                    self.longitude = address['geometry']['coordinates'][0]
+                    self.latitude = address['geometry']['coordinates'][1]
+                    self.municipality = address['properties']['kommunenavn']
+                    self.dawa_id = address['properties']['id']
+                    self.save()
+                    print("Opdateret for " + self.name)
+                    print("Updated coordinates for " + self.name)
+                    print("Updated municipality for " + self.name)
+                except Exception as error:
+                    print("Couldn't find coordinates for " + self.name)
+                    print("Error " + str(error))
+                    return None
+            else:
+                self.address_invalid = True
+                self.save()
+    # TODO: Move to dawa_data in utils

```

Comment:
det er vel som sådan ok det der har med person modellen står her. Men måske kunne der være en underfunktion der tog sig af selve adresse vask delen i utils. så kunne den kaldes andre steder hvis det bliver nødvendigt. Men det kan man jo altid lave når man rent faktisk har brug for den flere steder.

Reply:
Jeps, tænker vi kan gøre det når vi får brug for det. Jeg kan også godt flytte den nu hvis det er .